### PR TITLE
fix: Fixed punctuation on homepage.

### PR DESCRIPTION
**Pull request by mustafahasankhan**
Congratulations on the launch, much excited for this project.
Fixed a punctuation on home page.

### CODE REVIEW

**Review by nodeicode — APPROVED** [review]

**Comment by nodeicode** [comment]
changes are hard to read, tldr: this PR is changing 
```The goal of Bun is to run most of the worlds JavaScript outside of browsers```
to
```The goal of Bun is to run most of the world&apos;s JavaScript outside of browsers```

**Comment by mustafahasankhan** [comment]
> changes are hard to read, tldr: this PR is changing `The goal of Bun is to run most of the worlds JavaScript outside of browsers` to `The goal of Bun is to run most of the world&apos;s JavaScript outside of browsers`

Just added a missing apostrophe. `&apos;` denotes the apostrophe.

**Comment by tropicaaal** [comment]
This is changing the compiled version of the webpage and not the actual source of it. You should probably be making your changes to src/page.tsx rather than anything in `public`.

**Review by michellbrito — CHANGES_REQUESTED** [review]
Don't edit the index.html instead edit the `src/page.tsx` like @Tropix126 mentioned

**Comment by mustafahasankhan** [comment]
[Update] Added changes to source too.

File: packages/bun-landing/public/index.html [hunk 1]
```diff
@@ -928,7 +928,7 @@
...
.../polygon></g><g><polygon points="56,22 62,36 37,44"></polygon><polygon points="56,22 111,22 111,44 37,44 56,32" shape-rendering="crispEdges"></polygon><polygon points="116,95 97,117 90,104"></polygon><polygon points="116,95 100,104 97,117 42,117 42,95" shape-rendering="crispEdges"></polygon><polygon points="150,0 52,117 3,140 101,22"></polygon></g><g><polygon points="141,22 140,40 122,45"></polygon><polygon points="153,22 153,117 106,117 120,105 125,95 131,95 131,45 122,45 132,36 141,22" shape-rendering="crispEdges"></polygon><polygon points="125,95 130,110 106,117"></polygon></g></g><g fill="#121212"><g><polygon points="260,22 260,37 229,40 177,40 177,22" shape-rendering="crispEdges"></polygon><polygon points="260,37 207,99 207,103 176,103 229,40 229,37"></polygon><polygon points="261,99 261,117 176,117 176,103 206,99" shape-rendering="crispEdges"></polygon></g><rect x="272" y="22" shape-rendering="crispEdges" width="22" height="95"></rect><g><polygon points="394,67 394,106 376,106 376,81 360,70 346,67" shape-rendering="crispEdges"></polygon><polygon points="360,68 376,81 346,67"></polygon><path d="M394,106c-10.2,7.3-24,12-37.7,12c-29,0-51.1-20.8-51.1-48.3c0-27.3,22.5-48.1,52-48.1
```
Comment (michellbrito) [inline packages/bun-landing/public/index.html]: Undo the changes to the index.html so its changes is handle when its compiled from the page.tsx

**Review by michellbrito — DISMISSED** [review]

**Comment by FinnRG** [comment]
@mustafahasankhan Could you please undo the changes to the build output?

**Comment by mustafahasankhan** [comment]
> @mustafahasankhan Could you please undo the changes to the build output?

Done!

**Review by FinnRG — APPROVED** [review]

**Pull request merged**